src/main/index.ts

```diff
@@ -281,6 +281,7 @@ app.on("ready", async () => {
       });
 
     extensionLoader.initExtensions(extensions);
+    extensionLoader.getAvailableExtensionUpdates();
```

Comment:
I'm not really sure if this is good strategy for checking the updates. 

I would prefer if we created `checkForUpdate` method which takes `extension` as an argument. Then we could control each extension separately, and do the check right after initialization (if that's the requirement)?

Reply:
I will move this logic into `LensExtension` class so extension developers can extend the logic with their custom needs

Reply:
Hm, is that good thing? Do we want to expose this to Extension API?

Reply:
If someone creates extension for organization internal use, they can override update check and provide an update, for example, from their own servers.